sdcm/sct_runner.py

```diff
@@ -252,36 +323,164 @@ def create_image(self):
             self.tag_image(image_id=result["ImageId"], image_type=ImageType.general)
             LOGGER.info("Done.")
         else:
-            LOGGER.info("No need to copy SCT Runner image since it  already exists in '%s'.", self.region_name)
+            LOGGER.info("No need to copy SCT Runner image since it already exists in '%s'.", self.region_name)
 
-    def create_instance(self, test_id: str, test_duration: int, region_az: str):
+    def _get_base_image(self, image=None):
+        if image is None:
+            image = self.image
+        return image.id
+
+
+class GceSctRunner(SctRunner):
+    """Provisions and configures the SCT runner on GCE"""
+    BASE_IMAGE = "https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-2004-lts"
+    SOURCE_IMAGE_REGION = "us-east1"  # where the source Runner image will be created and copied to other regions
+    FAMILY = "sct-runner-image"
+    LOGIN_USER = "scylla-test"
+
+    def __init__(self, datacenter: str, availability_zone: str, cloud_provider: str = 'gce'):
+        self.availability_zone = availability_zone
+        self.gce_service = get_gce_service(datacenter)
+        self.gce_service_source = get_gce_service(self.SOURCE_IMAGE_REGION)
+        self._project = self.gce_service.ex_get_project()
+        self.PROJECT_NAME = self._project.name
+        super(GceSctRunner, self).__init__(cloud_provider=cloud_provider, region_name=datacenter)
```

Comment:
Make it in python3 style `super().__init__()` and move it to the very beginning

Reply:
fixed.